decidim-initiatives/app/commands/decidim/initiatives/vote_initiative.rb

```diff
@@ -39,10 +38,12 @@ def call
 
       private
 
+      attr_reader :form, :current_user
+
       def build_initiative_vote
-        @vote = @initiative.votes.build(
+        @vote = form.initiative.votes.build(
```

Comment:
Why this?

Reply:
I don't remember, the change is useless, I'll revert this